package.json

```diff
@@ -66,15 +66,14 @@
     "cssnano": "^3.10.0",
     "enzyme": "^3.0.0",
     "enzyme-adapter-react-16": "^1.1.1",
-    "eslint": "^3.12.0",
-    "eslint-config-airbnb": "13.0.0",
-    "eslint-config-airbnb-base": "10.0.1",
+    "eslint": "^6.0.0",
+    "eslint-config-policygenius": "^0.2.1",
     "eslint-import-resolver-webpack": "0.8.0",
-    "eslint-loader": "^1.7.1",
-    "eslint-plugin-import": "2.2.0",
-    "eslint-plugin-jsx-a11y": "2.2.3",
+    "eslint-loader": "^3.0.0",
+    "eslint-plugin-import": "^2.19.1",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-markdown": "https://github.com/jmcolella/eslint-plugin-markdown/tarball/master",
-    "eslint-plugin-react": "^6.8.0",
+    "eslint-plugin-react": "^7.17.0",
     "extract-text-webpack-plugin": "^2.1.0",
```

Comment:
I think you'll want to add a resolution here for js-yaml 3.13.1 since some other dependency tree is using a couple of the older versions